### PR TITLE
Backmerge: CDXML: When opening a saved file with text, the Font size enlarges #961

### PR DESCRIPTION
**Pull request by even1024**

### CODE REVIEW

File: api/tests/integration/ref/formats/ket_cdxml_ket.py.out [hunk 1]
```diff
@@ -81,15 +81,15 @@ agents.ket
...
..."atoms":[6,8]}]},"mol1":{"type":"molecule","atoms":[{"label":"C","location":[17.261444091796876,-5.758026123046875,0.0]},{"label":"C","location":[17.261444091796876,-6.75801944732666,0.0]},{"label":"C","location":[16.395389556884767,-7.258016109466553,0.0]},{"label":"C","location":[15.529335975646973,-6.75801944732666,0.0]},{"label":"C","location":[15.529335975646973,-5.758026123046875,0.0]},{"label":"C","location":[16.395389556884767,-5.258028984069824,0.0]},{"label":"C","location":[16.395389556884767,-4.258035659790039,0.0]},{"label":"C","location":[15.529335975646973,-3.7580387592315676,0.0]},{"label":"C","location":[14.663400650024414,-3.2580418586730959,0.0]},{"label":"O","location":[16.02933120727539,-2.8919837474823,0.0]},{"label":"C","location":[15.029337882995606,-4.624094009399414,0.0]}],"bonds":[{"type":1,"atoms":[0,1]},{"type":2,"atoms":[1,2]},{"type":1,"atoms":[2,3]},{"type":2,"atoms":[3,4]},{"type":1,"atoms":[4,5]},{"type":2,"atoms":[0,5]},{"type":1,"atoms":[5,6]},{"type":1,"atoms":[6,7]},{"type":1,"atoms":[7,8]},{"type":1,"atoms":[7,9]},{"type":1,"atoms":[7,10]}]},"mol2":{"type":"molecule","atoms":[{"label":"Cl","location":[10.475000381469727,-4.5,0.0]}],"bonds":[]}}
```
Comment (AlexanderSavelyev) [inline api/tests/integration/ref/formats/ket_cdxml_ket.py.out:92]: make ident for json and move output into a separate file

**Review by AlexanderSavelyev — COMMENTED** [review]
not accepted: need to split test file

**Review by AlexanderSavelyev — CHANGES_REQUESTED** [review]
split tests for cdx

File: api/tests/integration/ref/formats/ket_cdxml_ket.py.out (hunk 1 above)
Comment (even1024) [reply api/tests/integration/ref/formats/ket_cdxml_ket.py.out]: solved

**Review by AlexanderSavelyev — APPROVED** [review]

**Pull request merged**